tools/depends/native/TexturePacker/src/decoder/GifHelper.h

```diff
@@ -90,8 +87,7 @@ class GifHelper
   GifHelper();
```

Comment:
```suggestion
  GifHelper() = default;
```
This can be done when `m_gifFile` isn't a pointer anymore.

Reply:
nice